src/components/dialogs/network-modifications-dialog.js

```diff
@@ -36,107 +26,26 @@ const useStyles = makeStyles((theme) => ({
  * Dialog to select network modification to create
  * @param {Boolean} open Is the dialog open ?
  * @param {EventListener} onClose Event to close the dialog
- * @param network network from which to search for possible substations
- * @param selectedNodeUuid the currently selected tree node
+ * @param setDialogOpen to control the opening of dialogs
+ * @param dialogs the list of dialog
  */
 const NetworkModificationDialog = ({
     open,
     onClose,
-    network,
-    selectedNodeUuid,
-    workingNodeUuid,
+    setDialogOpen,
```

Comment:
rename this prop here to `onOpenDialog` ?
I know you assign it `setDialogOpen` in the parent but in here it should be called on*** I think...

Reply:
done